### ISSUE TITLE
TypeError: Cannot read property 'dateTime' of undefined (line 493, file "Helpers")

### ISSUE THREAD

**Issue by javedrahman** [derekantrican/GAS-ICS-Sync#133]
- Description of the problem - I am getting an error after upgrading to the new version.
- Screenshots 
![image](https://user-images.githubusercontent.com/11033953/83943762-66fef600-a7f6-11ea-81d2-bfc97fcb1de7.png)

- An input ics string that can reproduce the problem

You can see [this issue](https://github.com/derekantrican/GAS-ICS-Sync/issues/34) for an example of good information when submitting a bug

**Comment by jonas0b1011001** [comment]
Seems like a duplicate of #115 , without additional info i can only suggest the same change as mentioned there.

Did the same ics file work with the previous version?

**Comment by derekantrican** [comment]
@javedrahman Any update on this? What version are you running?

**Comment by javedrahman** [comment]
Hi. I am on version 5.3. Instead of using the Install method, I have created a timed trigger for startSync to run every 15 minutes. It actually seems to sync just fine but throws the error at the end so overall the script shows as Failed in My Executions.

I think there is an issue with an item in my calendar. because I tried changing the date range for the ICS export and now it does not always fail.
